proto/cosmos/vesting/v1beta1/vesting.proto

```diff
@@ -83,3 +84,23 @@ message PermanentLockedAccount {
 
   BaseVestingAccount base_vesting_account = 1 [(gogoproto.embed) = true];
 }
+
+// ClawbackVestingAccount implements the VestingAccount interface. It provides
+// an account that can hold contributions subject to "lockup" (like a
+// PeriodicVestingAccount), or vesting which is subject to clawback
+// of unvested tokens, or a combination (tokens vest, but are still locked).
+message ClawbackVestingAccount {
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  BaseVestingAccount base_vesting_account = 1 [(gogoproto.embed) = true];
+
+  // funder_address specifies the account which can perform clawback.
+  string                    funder_address = 2 [(gogoproto.moretags) = "yaml:\"funder_address\""];
+  google.protobuf.Timestamp start_time     = 3
+      [(gogoproto.moretags) = "yaml:\"start_time\"", (gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
Within the logic, we use it by converting this into `sdk.Int` by using Unix() method. 
An alternative way would be making this field `Int` in the first place, I currently don't have opinions abt which one's a better approach :/

Reply:
Yes, I was going to comment on this. There is no need for this to be a true timestamp. Let's make a change to have this field be an `Int`.

Reply:
Sorry, yeah `int64` 👍